### ISSUE TITLE
Update link colour on all alerts

### ISSUE THREAD

**Issue by vazquez** [mozilla/makerstrap#83]
Currently, `.alert-link` doesn't have the best colour contrast. This makes the text hard to read in alerts such as `.info` or `.success.
